src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -19,7 +20,7 @@ public class ListCommand extends Command {
             + "Example: " + COMMAND_WORD + " s/alphabet";
```

Comment:
Should update example to use `s/name` instead of `s/alphabet` now that it's changed.
I think there's no need to specify "Optional" parameter here. The convention for other Commands' MESSAGE_USAGE, as well as the UG, is that parameters in [square brackets] are optional without having to be explicitly stated.

Reply:
Ahh good spot, I missed that. Thanks!

But I just realised that this MESSAGE_USAGE is never used because there are no compulsory parameters. 

For instance, `add` will throw an error with its MESSAGE_USAGE if no index is inputed. However, list won't have such a case. In this case, should I remove the MESSAGE_USAGE or make the changes and leave for alignment with the other classes?

Reply:
Ah nevermind i think this could be a bug. Currently, `list asdfasdf n/name` will be taken as a `list` command. I think it should display an error message in this case. I shall make the appropriate changes

Reply:
I think it would be good to keep the MESSAGE_USAGE. Can be displayed in other cases like if the user gives some parameters which the command doesn't support, e.g. `list n/Bob`.

Though I now realise that none of the commands currently correct you if you add unsupported parameters, whether we should start to implement it is probably a whole other topic. For now, maybe we keep the MESSAGE_USAGE so at least we have it for the future? If we do, we can add in a comment that it's currently not used but have plans to add appropriate error messages.

Let me know what you think

Reply:
Yes I do agree that MESSAGE_USAGE should be used.

I think the commands currently do correct you. For instance, doing `edit 1 extraInvalid` would display the MESSAGE_USAGE instead. 

It would be the check for `!argMultimap.getPreamble().isEmpty()`. I have listed an example from the AddCommandParser class below.

``` Java
if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_ADDRESS, PREFIX_PHONE, PREFIX_EMAIL)
                || !argMultimap.getPreamble().isEmpty()) {
            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
        }
```

I have made the changes to the MESSAGE_USAGE as well as the error handling with regards to the unsupported parameters. 

Let me know what you think 😄

Reply:
I see what you mean. We can make that an issue and bring that up for further discussion in the future 👍